SwiftWisdom/Core/Foundation/URL+Extensions.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  URL+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Max Litteral on 7/17/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+extension URL: ExpressibleByStringLiteral {
+
+    public init(stringLiteral value: StaticString) {
+        self = URL(string: "\(value)")!
```

Comment:
Personally I would rather avoid this addition if it relies on a force unwrap for it's functionality. It's defeating the purpose of Apple's API having the `URL.init` be failable.